src/content/guides/asset-management.md

```diff
@@ -90,7 +87,7 @@ __webpack.config.js__
 
 Module loaders can be chained. Each loader in the chain applies transformations to the processed resource. A chain is executed in reverse order. The first loader passes its result (resource with applied transformations) to the next one, and so forth. Finally, webpack expects JavaScript to be returned by the last loader in the chain. 
 
-The above order of loaders should be maintained: 'style-loader' comes first and followed by 'css-loader'. If this convention is not followed, webpack is likely to throw errors.
+The above order of loaders should be maintained: `'style-loader'` comes first and followed by `'css-loader'`. If this convention is not followed, webpack is likely to throw errors.
```

Comment:
I think we should write the plugin to lint this, ideally we should always use "`" and provide link

Reply:
Single quote is preferred for code https://webpack.js.org/contribute/writers-guide/#quotation but there's no rule for text so far as I know.